### PR TITLE
switch to codecov

### PR DESCRIPTION
**Pull request by SenWu**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -50,8 +50,8 @@ Fonduer leverages the work of Emmental_ and Snorkel_.
    :target: https://github.com/HazyResearch/fonduer
 .. |CI-CD| image:: https://img.shields.io/github/workflow/status/HazyResearch/fonduer/ci.svg
    :target: https://github.com/HazyResearch/fonduer/actions
-.. |Coveralls| image:: https://img.shields.io/coveralls/github/HazyResearch/fonduer.svg
-   :target: https://coveralls.io/github/HazyResearch/fonduer
+.. |Codecov| image:: https://codecov.io/gh/HazyResearch/fonduer/branch/master/graph/badge.svg
```
Comment (lukehsiao) [inline README.rst:53]: For consistency, we can also use shields.io for this badge image.
```suggestion
.. |Codecov| image:: https://img.shields.io/codecov/c/github/HazyResearch/fonduer

**Comment by SenWu** [comment]
I removed the coverage test for all tests. So the coverage dropped to 82.5%

**Comment by lukehsiao** [comment]
At least one of the tests is always canceled. @SenWu, what's happening here?

**Comment by SenWu** [comment]
I don't know what's going on. Let me rerun all jobs.

**Pull request merged**